library/cwm/src/modules/CWM.rb

```diff
@@ -1026,6 +1060,30 @@ def validate_current_widgets(event)
     publish function: :ShowAndRunOrig, type: "symbol (list <string>, map <string, map <string, any>>, term, string, string, string, map)"
     publish function: :InitNull, type: "void (string)"
     publish function: :StoreNull, type: "void (string, map)"
+
+  private
+
+    def widgets_in_contents(contents)
```

Comment:
I see, Term being Enumerable is from 2015-01 but I run 13.2. Sorry for the noise!